### DISCUSSION TITLE
Cloud version in a refresh cycle

### DISCUSSION THREAD

**Discussion by Charlie23443** [Budibase/budibase#4209] [Help]
Hi all,

Has anybody experienced a constant refresh cycle of Budibase cloud version?

I'm running Chrome Version 97.0.4692.99 (Official Build) (64-bit).  I've tried clearing cookies etc, but no change.

However it is perfectly fine when using Edge.

**Comment by mjashanks** [comment]
Hey @Charlie23443 - at which point do you see this? Just after login? When you're building an app?

**Comment by Charlie23443** [comment]
Hi Michael,

It happens straight after logging in.

thanks,
Charlie
________________________________
